src/main/kotlin/game/racingcar/view/OutputView.kt

```diff
@@ -6,11 +6,11 @@ object OutputView {
 
     fun printOutputToConsole(racingCarPlayResult: RacingCarPlayResult) {
         println("실행 결과")
-        racingCarPlayResult.locationSnapshots.forEach { locationOfCars ->
+        racingCarPlayResult.carLocationSnapshots.forEach { locationOfCars ->
             locationOfCars.locations.forEach { snapshot ->
                 val (name, location) = snapshot
```

Comment:
data class는 람다 내에서 이렇게 구조 분해할 수 있어요 :)
```suggestion
            locationOfCars.locations.forEach { (name, location) ->
   
```